src/asm3/utils.py

```diff
@@ -1228,6 +1228,15 @@ def url(method, params):
             asm3.al.debug("strip invalid url '%s'" % l, "utils.fix_relative_document_uris", dbo)
     return s
 
+def fix_tinymce_uris(dbo: Database, s: str) -> str:
+    o = ""
+    for chunk in s.split("<"):
+        if chunk != '':
+            snippet = chunk.split(">")[0]
+            snippet = snippet.replace('&amp;', '&')
+            o += "<" + snippet + ">" + chunk.split(">")[1]
```

Comment:
This is a cheeky little function, like the use of split instead of regex groups. I have an irrational hatred of string concatenation though - make o a list and append to it instead. This pattern is less CPU intensive than doing lots of concatenation in a loop because you aren't creating a new immutable string for every iteration of the loop.

```
o = []
for ....
    do stuff
    o.append( value )
return "".join(o)
```